services/authnz/values-int.yaml

```diff
@@ -0,0 +1,87 @@
+authnz:
+  secrets:
+    enabled: False
+    # <generated flask secret>
+    flask_secret: "" 
+    # <generated cookie secret>
+    oauth2_proxy_cookie_secret: "" 
+    # <cilogon/OIDC secret>
+    oauth2_proxy_client_secret: "" 
+    # Generating a private key, then public key, then getting url-save version of it all
+    # echo "Generating Issuer Keypair... private.pem, public.pem"
+    # openssl genrsa -out private.pem 2048 2> /dev/null
+    # openssl rsa -in private.pem -outform PEM -pubout -out public.pem
+    # modulus_hex=$(openssl rsa -pubin -inform PEM -modulus -noout -in public.pem | sed 's/Modulus=//')
+    # modulus_urlsafe_b64=$(printf -- "$modulus_hex" | xxd -r -p | $b64 | sed 's/+/-/g;s/\//_/g;s/=//g')
+    signing_key: ""  # JWT Signing Key (private key)
+  
+  
+  # lsst-lsp-<instance>.ncsa.illinois.edu
+  host: lsst-lsp-int.ncsa.illinois.edu
+  
+  vault_secrets:
+    enabled: True
+    # secret/k8s_operator/<host>/jwt_authorizer
+    path: "secret/k8s_operator/lsst-lsp-int.ncsa.illinois.edu/jwt_authorizer"
+  
+  # Session length/Token expiration (in minutes)
+  session_length: 1440
+  
+  # Existing PVC for redis claim
+  # If empty, redis will use emptydir
+  redis_claim: "auth-int-volume-claim"
+  
+  
+  #  derived from public key of signing key - $(modulus_urlsafe_b64) from above
+  jwks_n: "3ALFUwJelCzi8FmGMOtKzbDj03mhGE_EslZ75_SpCgAeBbow44Wt3E-NqY1QGmadY-pRsl5G7ahtiAZscvijVx-1BfEyOx6HbXBZzDmVpNxELnagw-8OsH2JBJxeHjcbwZ7N-b3guwtHdSHLZmMxyBw_57OqYwfdAOjswFM4sUCwWo7ZkpGsVxeqQTflqMgJRBDKTtMk2Vjx6fwvP1x9uTxT9TIUseCngIYo64fVdQw0y_o6m0FrLM2N2F3E4KWkbUlTbisKcTWealKuX3OzUgDtpFHI-BUNgVakNgk0B4_G6mNnz1goZAn3JKZS8bAd5Au9x1O3ZPb9FkNn5xxN3w"
+  
+  # <client id from cilogon.org or OIDC provider>
+  oauth2_proxy_client_id: "cilogon:/client_id/6ca7b54ac075b65bccb9c885f9ba4a75"
+  
+  # Default capability required for user-based operations (like generating a token)
+  user_capability: "exec:user"
+  
+  # Default capability required for admin-based operations (like revoking a token, possibly)
+  admin_capability: "exec:admin"
+  
+  oauth2_proxy:
+    image: "lsstdm/oauth2_proxy:stable"
```

Comment:
Should use a version tag, not `stable`, so that it doesn't automatically upgrade.